js/modules/startup.js

```diff
@@ -0,0 +1,55 @@
+const is = require('@sindresorhus/is');
+
+const Errors = require('./types/errors');
+const Settings = require('./settings');
+
+
+exports.syncReadReceiptConfiguration = async ({
+  deviceId,
+  sendRequestConfigurationSyncMessage,
+  storage,
+}) => {
+  if (!is.string(deviceId)) {
```

Comment:
Thanks for the tip about `is`, @scottnonnenberg-signal! Will be even better once we use TypeScript ❤️